# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1318454588 25200
#      Wed Oct 12 14:23:08 2011 -0700
# Node ID f0daeb0eab45c8366d859780a14a6ae115317f65
# Parent  b68d95054947875dfcbf296ab8a206be440da971
gdbinit: Fix declaration of new show-octave-dbstack macro

* gdbinit: Fix declaration of new show-octave-dbstack macro

diff --git a/etc/gdbinit b/etc/gdbinit
--- a/etc/gdbinit
+++ b/etc/gdbinit
@@ -71,17 +71,17 @@ See also [display-dense-array] for showi
 of ordinary Octave arrays.
 end
 
 ############################################################
 
 ## Display Octave debugging stack 
 
 define show-octave-dbstack
-  call show_octave_dbstack
+  call show_octave_dbstack ()
 end
 
 document show-octave-dbstack
 Usage: show-octave-dbstack
 Display the contents of the current Octave debugging stack.
 
 This is the function stack that the Octave interpreter is processing
 and will be different from the C++ stack being debugged with gdb.
